tensorflow/examples/android/src/org/tensorflow/demo/CameraActivity.java

```diff
@@ -213,6 +387,7 @@ public boolean onKeyDown(final int keyCode, final KeyEvent event) {
     return super.onKeyDown(keyCode, event);
   }
 
+  protected abstract void processImageRGBbytes(byte[] luminance) ;
```

Comment:
I'm not sure I understand the purpose of this method as opposed to onImageAvailable() -- I can't see that luminance is ever filled in or used? Was this the missing functionality?